src/api/app/models/concerns/workflow_run_payload.rb

```diff
@@ -101,6 +102,18 @@ def path_with_namespace
     gitlab_path_with_namespace
   end
 
+  def label
+    pull_request_label
+  end
+
+  def pull_request_labeled?
+    github_pull_request_labeled?
+  end
+
+  def pull_request_unlabeled?
+    github_pull_request_unlabeled?
+  end
```

Comment:
All of this will barf it it's gitlab. And what about gitea?

Reply:
Nothing breaks for gitlab. The code to execute `label` filter just does not execute. 

I will address the naming convention change you mentioned

Reply:
You are still using a different naming scheme...

Reply:
The naming scheme has been adjusted, but for some reason, GitHub is not marking this comment as `outdated`.